### PR TITLE
 Merge amp-editor-blocks into amp-block-editor

### LINKED ISSUE #2333: Review shared amp-editor-blocks script
`amp-editor-blocks.js` is currently enqueued for both the regular block editor as well as the stories editor.

That script is rather small, and only applies a few filters to add some extra attributes and controls to existing blocks.

As pointed out in https://github.com/ampproject/amp-wp/pull/2331#discussion_r284780247, most if not all of the controls added by  that script do not make sense in a stories context. For example the AMP layout, noloading, or lightbox controls.

We need to go through them and see whether we need to move them over to `amp-block-editor.js` (which doesn't load for stories). Perhaps we can even get rid of `amp-editor-blocks.js` so that everything is in either `block-editor.js` or `amp-stories.js`.

Tagging @westonruter @kienstra as you probably know these controls better than I.

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #2333.

### CODE REVIEW

File: assets/src/block-editor/helpers/index.js [hunk 1]
```diff
@@ -133,7 +133,8 @@ export const addAMPAttributes = ( settings, name ) => {
 			attribute: 'max-font-size',
 		};
 		settings.attributes.height = {
-			default: 'core/image' === name ? 200 : 50,
+			// Needs to be higher than the maximum font size, which defaults to MAX_FONT_SIZE
+			default: 'core/image' === name ? 200 : 80,
```
Comment (miina) [inline assets/src/block-editor/helpers/index.js]: Should we use the constant here instead then, e.g. `MAX_FONT_SIZE + ...`?

File: includes/admin/class-amp-editor-blocks.php [hunk 2]
```diff
@@ -120,49 +119,6 @@ public function whitelist_block_atts_in_wp_kses_allowed_html( $tags, $context )
...
-		wp_enqueue_script(
-			$script_handle,
-			amp_get_asset_url( 'js/amp-editor-blocks.js' ),
-			$script_dependencies,
-			AMP__VERSION,
-			true
-		);
-
-		if ( function_exists( 'wp_set_script_translations' ) ) {
-			wp_set_script_translations( $script_handle, 'amp' );
```
Comment (miina) [inline includes/admin/class-amp-editor-blocks.php]: Curious to know -- is `wp_set_script_translations` not needed anymore?

Comment (swissspidy) [reply includes/admin/class-amp-editor-blocks.php]: This `wp_set_script_translations` call here is for `amp-editor-blocks.js`, which this PR removes 🙂 

`wp_set_script_translations` for the one and only `amp-block-editor.js` script that's left is still in `AMP_Post_Meta_Box`.

Comment (miina) [reply includes/admin/class-amp-editor-blocks.php]: Ah, makes sense, thanks!

File: assets/src/block-editor/helpers/index.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/block-editor/helpers/index.js]: Fixed in 0e3fbe3f0ff8d66e7a78ecc7368c733189935e9a

**Comment by swissspidy** [comment]
^ The PR build failure is just a Travis hiccup.

**Pull request merged**